### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by bouaik**
- created form to add books
- books are stored in local storage
-  books can be deleted from the dom

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                        <div class="input-group-prepend">
+                            <span class="input-group-text"><i class="fas fa-user"></i></span>
+                        </div>
+                        <input type="text" class="form-control text-capitalize author" placeholder="author" id="author">
+                    </div>
+                    <div class="input-group my-3">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text"><i class="fas fa-file-alt"></i></span>
+                        </div>
+                        <input type="number" class="form-control text-capitalize pages" placeholder="number of pages"
```
Comment (mcrd25) [inline index.html:37]: [OPTIONAL] Consider even adding required attribute for simple validation

File: index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                        <div class="input-group-prepend">
+                            <span class="input-group-text"><i class="fas fa-book"></i></span>
+                        </div>
+                        <input type="text" class="form-control text-capitalize title" id="title" placeholder="title">
+                    </div>
+                    <div class="input-group my-3">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text"><i class="fas fa-user"></i></span>
+                        </div>
+                        <input type="text" class="form-control text-capitalize author" placeholder="author" id="author">
```
Comment (mcrd25) [inline index.html:31]: [OPTIONAL] Consider even adding required attribute for simple validation

File: index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    <div class="container">
+        <div class="row">
+            <div class="col-11 mx-auto text-center my-5">
+                <form class="my-5" id="customer-form">
+                    <h3 class="text-capitalize">My Library</h3>
+                    <div class="input-group my-3">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text"><i class="fas fa-book"></i></span>
+                        </div>
+                        <input type="text" class="form-control text-capitalize title" id="title" placeholder="title">
```
Comment (mcrd25) [inline index.html:25]: [OPTIONAL] Consider even adding required attribute for simple validation

File: index.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                            <input type="radio" id="customRadioInline1" name="customRadioInline1"
+                                class="custom-control-input" value="yes">
+                            <label class="custom-control-label" for="customRadioInline1">Yes</label>
+                        </div>
+                        <div class="custom-control custom-radio custom-control-inline pt-1">
+                            <input type="radio" id="customRadioInline2" name="customRadioInline1"
+                                class="custom-control-input" value="no">
+                            <label class="custom-control-label" for="customRadioInline2">No</label>
+                        </div>
+                    </div>
```
Comment (mcrd25) [inline index.html:54]: [OPTIONAL]
Make one of these a default value by marking it checked. I suggest the `customeRadioInline2` as default checked value.

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required :mag: 

Hi @bouaik :wave: 

It's cool that you chose to do use local storage! The project looks fine! :slightly_smiling_face: 

You have only one key feature missing:
#### Required changes
- You ought to have a toggle for whether or not a book is read. See [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/toggle_requirement.png) from Odin.

#### Some OPTIONAL changes
- Consider adding some validation to your form. As it stands I can leave all parts blank.


***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya Douglas](mayacdouglas.com)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by uryelah — APPROVED** [review]
Hi **Lhoussaine Bouaik**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job adding that toggle!

Ps: Remember to[ capitalize your commits in the next projects.](https://chris.beams.io/posts/git-commit/)

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**


---------

    
    
**Keep being awesome devs and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**